frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -58,6 +64,30 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
     });
   };
 
+  const checked = !!instances.filter(i => selectiveDestroy?.includes(i.id))
+    .length;
+
+  const indeterminate =
+    instances.filter(i => selectiveDestroy?.includes(i.id)).length !==
+      instances.length && checked;
+
+  const selectGroup = () => {
+    if (checked) {
+      /* if (indeterminate) {
+        const tmp = instances.filter(i => !selectiveDestroy?.includes(i.id));
+        tmp.forEach(i => selectToDestroy!(i.id));
+      } else {
+        const tmp = instances.filter(i => selectiveDestroy?.includes(i.id));
+        tmp.forEach(i => selectToDestroy!(i.id));
+      } */
```

Comment:
Remember to purge this if unused :)

Reply:
yes, i'm only waiting that someone else besides me tries and tests the deploy staging. only to be sure the changes hasn't created any problem